### PR TITLE
Sidecar buttons

### PR DESCRIPTION
**Pull request by Scott-Guinn**
Styling changes made to the buttons and Apple Progress Bar

### CODE REVIEW

File: components/CalorieTracker.jsx [hunk 1]
```diff
@@ -18,22 +18,24 @@ const CalorieTracker = () => {
...
-        `}
-        </style>
         <ProgressBar
+          style={{
+            height: '100%',
+            width: '190px',
+          }}
           className="calorie-progress-bar"
-          variant='success'
+          variant='danger'
```
Comment (emmaknor) [inline components/CalorieTracker.jsx:27]: scary

File: components/CalorieTracker.jsx [hunk 2]
```diff
@@ -18,22 +18,24 @@ const CalorieTracker = () => {
...
           now={progress}
         />
         <Apple />
       </div>
-      <button
+
+      <Button
+        style={{
+          position: 'absolute',
+          zIndex: '2', bottom: '2%',
```
Comment (emmaknor) [inline components/CalorieTracker.jsx:36]: I haven't used a z-index of 2 before. Are there two layers elements you are stacking this button on?

File: components/dailyTracker.jsx [hunk 3]
```diff
@@ -19,65 +19,89 @@ const DailyTracker = () => {
...
+              }}
+              title='calorie-btn'>
+              <img src='/restaurant.png'
+                style={{
+                  height: '28px',
+                  width: '28px',
+                }}/>
+            </Button>
+          </Col>
+          <Col xs={6}>
```
Comment (emmaknor) [inline components/dailyTracker.jsx:49]: any reason for only adding responsive design to xs screens? is the sizing irrelevant for this component on larger screens?

**Review by emmaknor — APPROVED** [review]
Great work Scott! Thank you for taking on styling the data entry buttons. I appreciate it!

File: components/CalorieTracker.jsx (hunk 1 above)
Comment (mmoren01) [reply components/CalorieTracker.jsx:27]: You have no idea @emmaknor

File: components/CalorieTracker.jsx (hunk 2 above)
Comment (Scott-Guinn) [reply components/CalorieTracker.jsx:36]: this was just to make sure the test button was on top of everything but this whole button will be removed now that demo is over

File: components/dailyTracker.jsx (hunk 3 above)
Comment (Scott-Guinn) [reply components/dailyTracker.jsx:49]: So my understanding is that this sets the middle column to be half of the available space (6/12) leaving 3/12 for each sidecar button.

On larger screens it looks good so I didn't touch any of the other ones but not totally sure how to use this so open to suggestions if you think there's an improvement to be had.

Comment (mmoren01) [reply components/dailyTracker.jsx:49]: I think this is fine for MVP. For a polished product we should go back and add responsive design for more screen sizes.

**Review by mmoren01 — APPROVED** [review]

**Pull request merged**